source/release-notes/3.6.txt

```diff
@@ -199,23 +199,20 @@ Multi-Element Array Updates
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 MongoDB 3.6 adds the following new positional operators for update
-operations:
+operations on arrays and nested arrays:
 
-- The all positional :update:`$[]` operator can support updating all
-  elements in an array.The all positional :update:`$[]` operator can
-  support updating nested arrays.
+- The all positional :update:`$[]` operator updatings all elements in
```

Comment:
minor -- updatings -> updates

Reply:
Oops .. missed that from previous edit. Will correct and push an update to this branch.

Reply:
@kay-kim Updated with #f1c8ece